pynear/include/BKTree.hpp

```diff
@@ -68,15 +68,16 @@ template <typename key_t, typename distance_t, typename metric> class BKTree {
         }
     }
 
-    std::tuple<std::vector<distance_t>, std::vector<key_t>> find(key_t key, distance_t threshold) {
+    std::tuple<std::vector<index_t>, std::vector<distance_t>, std::vector<key_t>> find(key_t key, distance_t threshold) {
```

Comment:
Im trying to add some docs in the functions in my open PR. We still need to add a few, but if possible, can you try to document a bit the function.  If you are ok with it, we can use the given format:
```cpp
/*
* Function description and explanation.
* Args:
* arg1: description
*/
```
We can also do this in a separate PR later on if you think its better. Thanks.

Reply:
Is there any standard for cpp we can follow? I know there are a few for python. But yes, maybe we can add this in a separate PR.